starport/pkg/chaincmd/runner/chain.go

```diff
@@ -0,0 +1,78 @@
+package chaincmdrunner
+
+import (
+	"bytes"
+	"context"
+	"regexp"
+	"strings"
+
+	"github.com/tendermint/starport/starport/pkg/chaincmd"
+)
+
+// Start starts the blockchain.
+func (r Runner) Start(ctx context.Context, args ...string) error {
+	return r.run(ctx, runOptions{longRunning: true}, r.cc.StartCommand(args...))
+}
+
+// LaunchpadStartRestServer start launchpad rest server.
+func (r Runner) LaunchpadStartRestServer(ctx context.Context, apiAddress, rpcAddress string) error {
+	return r.run(ctx, runOptions{longRunning: true}, r.cc.LaunchpadRestServerCommand(apiAddress, rpcAddress))
+}
+
+// Init inits the blockchain.
+func (r Runner) Init(ctx context.Context, moniker string) error {
+	return r.run(ctx, runOptions{}, r.cc.InitCommand(moniker))
+}
+
+// KeyValue holds a key, value pair.
+type KeyValue struct {
+	key   string
+	value string
+}
+
+// KV returns a new key, value pair.
+func KV(key, value string) KeyValue {
```

Comment:
```suggestion
func NewKeyValue(key, value string) KeyValue {
```

`New[TypeName]` is kind of a convention isn't it?

Reply:
solved by 7584034.